### DISCUSSION TITLE
LS-DYNA unrecognized option, error ID 100210

### DISCUSSION THREAD

**Discussion by mfontMB** [OpenRadioss/OpenRadioss#3148] [Q&A]
Hello everyone,

I’m quite new to using OpenRadioss, and I must say I’m truly impressed with the remarkable work behind its development. I’m incredibly grateful for the opportunity to use such a powerful tool and I sincerely hope to contribute to the growth of this community.

At the moment, I’m successfully running the Camry and Zug tests available on the DYNA examples site. However, I’m now working on a more complex model that includes certain cards that I’ve noticed are not listed in the Altair Reference Guide. That said, when I run a grep command in the ./hm_cfg_files/config/CFG directory, I find that these cards are present, which leads me to believe they might be implemented.

I would greatly appreciate it if anyone could clarify whether the presence of a card in the CFG directory means that it can be converted from DYNA to RADIOSS?

My questions are as follows:

Are these cards indeed implemented in OpenRadioss?
If so, how can I make them runnable?
If they are not implemented, would it be advisable to look for alternative cards with similar functionality?
I’m really grateful in advance for any guidance or insights from the community. Below, I’ve included an example and a full list of the errors I’m encountering at the end of this message.

Thank you so much for your time and support!

For instance:
Error I get:
ERROR ID : 100210
** ERROR IN INPUT OPTIONS
DESCRIPTION :  
-- BLOCK: *MAT_VISCOUS_FOAM
-- LINE: *MAT_VISCOUS_FOAM
Unrecognized option: *MAT_VISCOUS_FOAM

Grep command:
myPC:~/OpenRadioss/hm_cfg_files/config/CFG$ grep -r "MAT_VISCOUS_FOAM"
Keyword971_R9.3/data_hierarchy.cfg:         TITLE      = "062 MAT_VISCOUS_FOAM";
Keyword971_R9.3/data_hierarchy.cfg:         USER_NAMES = (MAT_VISCOUS_FOAM,MAT_062,MATL62);
Keyword971_R8.0/data_hierarchy.cfg:         TITLE      = "062 MAT_VISCOUS_FOAM";
Keyword971_R8.0/data_hierarchy.cfg:         USER_NAMES = (MAT_VISCOUS_FOAM,MAT_062,MATL62);
Keyword971_R6.1/data_hierarchy.cfg:         TITLE      = "062 MAT_VISCOUS_FOAM";
Keyword971_R6.1/data_hierarchy.cfg:         USER_NAMES = (MAT_VISCOUS_FOAM,MAT_062,MATL62);
Keyword971_R12.0/data_hierarchy.cfg:         TITLE      = "062 MAT_VISCOUS_FOAM";
Keyword971_R12.0/data_hierarchy.cfg:         USER_NAMES = (MAT_VISCOUS_FOAM,MAT_062,MATL62);
Keyword971_R7.1/data_hierarchy.cfg:         TITLE      = "062 MAT_VISCOUS_FOAM";
Keyword971_R7.1/data_hierarchy.cfg:         USER_NAMES = (MAT_VISCOUS_FOAM,MAT_062,MATL62);
Keyword971_R10.1/data_hierarchy.cfg:         TITLE      = "062 MAT_VISCOUS_FOAM";
Keyword971_R10.1/data_hierarchy.cfg:         USER_NAMES = (MAT_VISCOUS_FOAM,MAT_062,MATL62);
Keyword971_R13.0/data_hierarchy.cfg:         TITLE      = "062 MAT_VISCOUS_FOAM";
Keyword971_R13.0/data_hierarchy.cfg:         USER_NAMES = (MAT_VISCOUS_FOAM,MAT_062,MATL62);
Keyword971_R11.2/data_hierarchy.cfg:         TITLE      = "062 MAT_VISCOUS_FOAM";
Keyword971_R11.2/data_hierarchy.cfg:         USER_NAMES = (MAT_VISCOUS_FOAM,MAT_062,MATL62);
Keyword971/data_hierarchy.cfg:         TITLE      = "062 MAT_VISCOUS_FOAM";
Keyword971/data_hierarchy.cfg:         USER_NAMES = (MAT_VISCOUS_FOAM,MAT_062,MATL62);
Keyword971/MAT/mat_062.cfg:     ASSIGN(KEYWORD_STR, "*MAT_VISCOUS_FOAM");
Keyword971_R11.1/data_hierarchy.cfg:         TITLE      = "062 MAT_VISCOUS_FOAM";
Keyword971_R11.1/data_hierarchy.cfg:         USER_NAMES = (MAT_VISCOUS_FOAM,MAT_062,MATL62);

List of errors:
ERROR ID : 100210
** ERROR IN INPUT OPTIONS
DESCRIPTION :  
-- BLOCK: *CONTACT_RIGID_BODY_ONE_WAY_TO_RIGID_BODY
-- LINE: *CONTACT_RIGID_BODY_ONE_WAY_TO_RIGID_BODY
Unrecognized option: *CONTACT_RIGID_BODY_ONE_WAY_TO_RIGID_BODY

ERROR ID : 100210
** ERROR IN INPUT OPTIONS
DESCRIPTION :  
-- BLOCK: *CONTACT_RIGID_BODY_ONE_WAY_TO_RIGID_BODY
-- LINE: *CONTACT_RIGID_BODY_ONE_WAY_TO_RIGID_BODY
Unrecognized option: *CONTACT_RIGID_BODY_ONE_WAY_TO_RIGID_BODY

WARNING ID : 100213
** WARNING IN INPUT OPTIONS
DESCRIPTION :  
-- BLOCK: *CONTROL_SHELL
-- LINE: 0         0         0         0         0         1         1
unsupported field exists at the end of line
SOLUTION :     
    Please check input format and fields description defined in the Reference Guide.

ERROR ID : 100210
** ERROR IN INPUT OPTIONS
DESCRIPTION :  
-- BLOCK: *MAT_VISCOUS_FOAM
-- LINE: *MAT_VISCOUS_FOAM
Unrecognized option: *MAT_VISCOUS_FOAM

ERROR ID : 100210
** ERROR IN INPUT OPTIONS
DESCRIPTION :  
-- BLOCK: *MAT_VISCOUS_FOAM
-- LINE: *MAT_VISCOUS_FOAM
Unrecognized option: *MAT_VISCOUS_FOAM

WARNING ID : 100217
** WARNING IN INPUT OPTIONS
DESCRIPTION :  
-- BLOCK: *SECTION_SHELL
-- LINE: *MAT_FABRIC
card is missing
SOLUTION :     
Please check input format and fields description defined in the Reference Guide.

WARNING ID : 100214
** WARNING IN INPUT OPTIONS
DESCRIPTION :  
-- BLOCK: *MAT_FABRIC
-- LINE: 2060000 1.0000E-6  2.000000  2.000000  2.000000  0.300000  0.300000  0.300000
unsupported field exists
SOLUTION :     
    Please check input format and fields description defined in the Reference Guide.

WARNING ID : 100214
** WARNING IN INPUT OPTIONS
DESCRIPTION :  
-- BLOCK: *MAT_FABRIC
-- LINE: 0.000     0.000     0.000  1.000000
unsupported field exists
SOLUTION :     
    Please check input format and fields description defined in the Reference Guide.

ERROR ID : 100210
** ERROR IN INPUT OPTIONS
DESCRIPTION :  
-- BLOCK: *ELEMENT_SEATBELT_RETRACTOR
-- LINE: *ELEMENT_SEATBELT_RETRACTOR
Unrecognized option: *ELEMENT_SEATBELT_RETRACTOR

ERROR ID : 100210
** ERROR IN INPUT OPTIONS
DESCRIPTION :  
-- BLOCK: *ELEMENT_SEATBELT_SENSOR
-- LINE: *ELEMENT_SEATBELT_SENSOR
Unrecognized option: *ELEMENT_SEATBELT_SENSOR

ERROR ID :   2000
** ERROR WHILE READING INPUT FILE
DESCRIPTION :  
   6 ERRORS FOUND

 RADIOSS STOP DUE TO INPUT ERROR

**Comment by sebastienVilleneuve** [comment]
Hi @mfontMB 


### _**Are**_ these cards indeed implemented in OpenRadioss?
You're right , there is reference MAT_VISCOUS_FOAM in datahierarchy , but if you double check , you will see that for the moment , it is commented 
![image](https://github.com/user-attachments/assets/9f701e31-1b85-4785-957f-67e02f59a727)
( c syntax /*   */  )

ELEMENT_SEATBELT_SENSOR , ELEMENT_SEATBELT_RETRACTOR, CONTACT_RIGID_BODY_ONE_WAY_TO_RIGID_BODY 
are not supported too 


### **_If so, how can I make them runnable?_**
It will need some development on our side




### **_If they are not implemented, would it be advisable to look for alternative cards with similar functionality?_**
it is difficult to give you a general solution, but can you send us the MAT card , we will make some proposal .




Thanks,
Sébastien

**Reply by jan-skoda** [reply]
Hallo,
I would like to know, if there is any progress in the *mat_viscous_foam recognition?
I tried to play with THUMS dummy. There are unrecognized LS-Dyna cards:
*DATABASE_CURVOUT
*MAT_VISCOUS_FOAM
*CONTACT_AUTOMATIC_BEAMS_TO_SURFACE_ID

Is there any possibility to workaround?

Thanks
Jan Skoda
